ambry-frontend/src/main/java/com/github/ambry/frontend/NamedBlobsCleanupRunner.java

```diff
@@ -57,6 +57,7 @@ public void run() {
       List<StaleNamedBlob> batchStaleBlobs = Collections.emptyList();
       NamedBlobDb.StaleBlobsWithLatestBlobName staleBlobsWithLatestBlobName;
       for (Container container : combinedContainers) {
+        logger.info("Started the cleaner for container: {}", container.getId());
```

Comment:
let's move this log under the if statement below, we don't need this message for a container that disabled named blobs.